.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
```

Comment:
Instead of switching back to v1, you could also set
```yaml
with:
    fetch-depth: 0  # disable shallow clones
```
(see https://github.com/actions/checkout)

This should be more forward-compatible.

Reply:
Yes, it's a better option.